rpm/util-linux.spec

```diff
@@ -198,6 +198,26 @@ Obsoletes: %{name}-docs
 %description doc
 Man and info pages for %{name}.
 
+%package systemd-fstrim
+Summary:   Systemd unit files for %{name}
+Group:     System
+Requires:  %{name} = %{version}-%{release}
+Requires(post): systemd
+BuildRequires:  systemd-devel
+
+%description systemd-fstrim
+Systemd unit files for fstrim
+
+%package systemd-uuidd
+Summary:   Systemd unit files for %{name}
+Group:     System
+Requires:  %{name} = %{version}-%{release}
+Requires:  uuidd
+Requires(post): systemd
+BuildRequires:  systemd-devel
```

Comment:
`BuildRequires` should be add once in the beginning of the spec as it's not sub-package specific.
And it's better to use pkgconfig:
```
BuildRequires: pkgconfig(systemd)
```

Reply:
Requires(post) is fine but I agree use pkgconfig please.

Reply:
O.K., done.